doc/source/rllib/external-envs.rst

```diff
@@ -153,7 +153,7 @@ Responses: Server → Client
 
 - **``SET_STATE``**
 
-  - Example: ``{"type": "PONG"}``
+  - Example: ``{"type": "SET_STATE", "weights_seq_no": 123, "onnx_file": "... [base64 encoded ONNX file] ..."}``
```

Comment:
No idea about this change. Needs tech validation.

Reply:
Great catch! This looks good.